platform/mbed_retarget.cpp

```diff
@@ -1255,46 +1233,29 @@ extern "C" WEAK void __cxa_pure_virtual(void)
 // SP.  This make it compatible with RTX RTOS thread stacks.
 #if defined(TOOLCHAIN_GCC_ARM) || defined(TOOLCHAIN_GCC_CR)
 
-#if defined(TARGET_CORTEX_A)
-extern "C" uint32_t  __HeapLimit;
+#if !defined(HEAP_START)
```

Comment:
This can be remove I we assume that `__end__` and `__HeapLimit` are always defined.

Reply:
Will leave it as is to explore the heap in multiple RAM banks, once that is implemented can remove this